README.md

```diff
@@ -35,6 +35,22 @@ $ echidna-test examples/solidity/basic/flags.sol
 
 Echidna should find a a call sequence that falisfies `echidna_sometimesfalse` and should be unable to find a falsifying input for `echidna_alwaystrue`.
 
+### Truffle integration
+
+Echidna can be used to test contracts compiled with [Truffle](https://truffleframework.com/) using [crytic-compile](https://github.com/crytic/crytic-compile). For instance,
+we can uncover an integer overflow in the [Metacoin Truffle box](https://github.com/truffle-box/metacoin-box) using a
+[contract with Echidna properties to test](examples/solidity/truffle/metacoin/contracts/MetaCoinEchidna.sol):
+
+```
+$ cd examples/solidity/truffle/metacoin
+$ crytic-compile --export-format solc .
+$ echidna-test crytic-export/combined_solc.json $(pwd)/contracts/MetaCoinEchidna.sol:TEST
```

Comment:
Could we remove the need for `$(pwd)/contracts/`?

Reply:
Done.